vmdb/spec/lib/extensions/ar_virtual_spec.rb

```diff
@@ -631,47 +631,42 @@ def self.reflections; super.merge(:ref2 => OpenStruct.new(:name => :ref2, :optio
 
     context "virtual column" do
       it "as Symbol" do
-        lambda { Vm.all(:include => :platform) }.should_not raise_error
+        lambda { Vm.includes(:platform).load }.should_not raise_error
```

Comment:
@imtayadeway that doesn't seem to apply here, because these are negative assertions